dipy/io/gradients.py

```diff
@@ -62,11 +64,16 @@ def read_bvals_bvecs(fbvals, fbvecs):
     if bvecs is None:
         return bvals, bvecs
 
-    if min(bvecs.shape) != 3:
+    if 3 not in bvecs.shape:
         raise IOError('bvec file should have three rows')
     if bvecs.ndim != 2:
-        raise IOError('bvec file should be saved as a two dimensional array')
-    if bvecs.shape[1] > bvecs.shape[0]:
+        bvecs = bvecs[None, ...]
+        bvals = bvals[None, ...]
+        msg = "Only 1 direction detected on your bvec file. For diffusion "
+        msg += "dataset, it is recommended to have minimum 3 directions."
+        msg += "You may have problem during the reconstruction step."
```

Comment:
Maybe ?
```
msg = "Detected only 1 direction on your bvec file. For diffusion "
msg += "datasets, it is recommended to have at least 3 directions."
msg += "You may have problems during the reconstruction step."
```

Reply:
" (...) problems (...)."

Reply:
Thanks, for sure I prefer your version. updated